packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -92,18 +92,20 @@ export class GitHubSource extends BuildSource {
 }
 
 /**
- * GitHub Enterprice Source definition for a CodeBuild project
+ * GitHub Enterprise Source definition for a CodeBuild project
  */
 export class GitHubEnterpriseSource extends BuildSource {
-  constructor(private readonly cloneUrl: string) {
+  constructor(private readonly cloneUrl: string, private readonly oauthToken: any) {
```

Comment:
Was not sure if I should leave make oauthToken optional?
```typescript
constructor(private readonly cloneUrl: string, private readonly oauthToken?: any) {
```

I was mostly copying how it was done w/ `GitHubSource` but now this is breaking change I believe.

Reply:
@skinny85 - should the token be `any`?

Reply:
Going by the [GitHub CodePipeline Action example](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-codepipeline/lib/github-source-action.ts#L44), it seems like `oauthToken` should be a `cdk.Secret`.

Reply:
@skinny85 @moofish32 I was going off. https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-codebuild/lib/source.ts#L79

Do you want me to change both to `cdk.Secret`?